src/components/views/location/ShareDialogButtons.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+
+import AccessibleButton from '../elements/AccessibleButton';
+import { Icon as BackIcon } from '../../../../res/img/element-icons/caret-left.svg';
+import { Icon as CloseIcon } from '../../../../res/img/element-icons/cancel-rounded.svg';
+
+interface Props {
+    onCancel: () => void;
+    onBack: () => void;
+    displayBack?: boolean;
+}
+
+const ShareDialogButtons: React.FC<Props> = ({ onBack, onCancel, displayBack }) => {
+    return <div className='mx_ShareDialogButtons'>
+        { displayBack &&
+            <AccessibleButton
+                className="mx_ShareDialogButtons_button left"
+                data-test-id='share-dialog-buttons-back'
+                onClick={onBack}
+                element='button'>
```

Comment:
```suggestion
                className="mx_ShareDialogButtons_button left"
                data-test-id='share-dialog-buttons-back'
                onClick={onBack}
                element='button'
            >
```

and could we stay consistent with one type of quotes in the same block of code please